contracts/src/plugin/extensions/proposal/Proposal.sol

```diff
@@ -2,82 +2,35 @@
-    using Counters for Counters.Counter;
-
-    /// @notice The incremental ID for proposals and executions.
-    Counters.Counter private proposalCounter;
-
     /// @inheritdoc IProposal
-    function proposalCount() public view override returns (uint256) {
-        return proposalCounter.current();
+    function proposalCount() public pure override returns (uint256) {
+        return type(uint256).max;
     }
 
     /// @notice Checks if this or the parent contract supports an interface by its ID.
     /// @param _interfaceId The ID of the interface.
     /// @return Returns `true` if the interface is supported.
     function supportsInterface(bytes4 _interfaceId) public view virtual override returns (bool) {
-        return _interfaceId == type(IProposal).interfaceId || super.supportsInterface(_interfaceId);
-    }
-
-    /// @notice Creates a proposal ID.
-    /// @return proposalId The proposal ID.
-    function _createProposalId() internal returns (uint256 proposalId) {
-        proposalId = proposalCount();
-        proposalCounter.increment();
-    }
-
-    /// @notice Internal function to create a proposal.
-    /// @param _metadata The proposal metadata.
-    /// @param _startDate The start date of the proposal in seconds.
-    /// @param _endDate The end date of the proposal in seconds.
-    /// @param _allowFailureMap A bitmap allowing the proposal to succeed, even if individual actions might revert. If the bit at index `i` is 1, the proposal succeeds even if the `i`th action reverts. A failure map value of 0 requires every action to not revert.
-    /// @param _actions The actions that will be executed after the proposal passes.
-    /// @return proposalId The ID of the proposal.
-    function _createProposal(
-        address _creator,
-        bytes calldata _metadata,
-        uint64 _startDate,
-        uint64 _endDate,
-        IDAO.Action[] calldata _actions,
-        uint256 _allowFailureMap
-    ) internal virtual returns (uint256 proposalId) {
-        proposalId = _createProposalId();
-
-        emit ProposalCreated({
-            proposalId: proposalId,
-            creator: _creator,
-            metadata: _metadata,
-            startDate: _startDate,
-            endDate: _endDate,
-            actions: _actions,
-            allowFailureMap: _allowFailureMap
-        });
-    }
-
-    /// @notice Internal function to execute a proposal.
-    /// @param _proposalId The ID of the proposal to be executed.
-    /// @param _actions The array of actions to be executed.
-    /// @param _allowFailureMap A bitmap allowing the proposal to succeed, even if individual actions might revert. If the bit at index `i` is 1, the proposal succeeds even if the `i`th action reverts. A failure map value of 0 requires every action to not revert.
-    /// @return execResults The array with the results of the executed actions.
-    /// @return failureMap The failure map encoding which actions have failed.
-    function _executeProposal(
```

Comment:
Assuming that this is removed because plugins will be targeting an executor, rather than calling `execute()` locally